services/modrinth/modrinth-base.js

```diff
@@ -0,0 +1,38 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+import { nonNegativeInteger } from '../validators.js'
+
+const projectSchema = Joi.object({
+  downloads: nonNegativeInteger,
+  followers: nonNegativeInteger,
+}).required()
+
+const versionSchema = Joi.array()
+  .items(
+    Joi.object({
+      version_number: Joi.string(),
+    })
+  )
+  .required()
```

Comment:
The schema should be

```suggestion
const versionSchema = Joi.array()
  .items(
    Joi.object({
      version_number: Joi.string().required(),
    }).required()
  )
  .required()
```

We want to make sure that there is at least one item in the array and the `version_number` peoperty is required.